addons/point_of_sale/static/src/js/models.js

```diff
@@ -1460,6 +1460,13 @@ export class Product extends PosModel {
         }
         return this.pos.units_by_id[unit_id];
     }
+    is_pricelist_item_usable(item, context) {
```

Comment:
Since this concern master, would you mind using camelCase for naming ?
```suggestion
    isPricelistItemUsable(item, context) {
```
We're slowly uniforming styling